README.md

```diff
@@ -13,88 +16,45 @@ npm i -g good-first-issue
 good-first-issue <project>
 ```
 
+### As Module
+
+```js
+const gfi = require('good-first-issue')
+
+// project should be from the below projects table
+gfi(project, (err, issues) => {
+  // typeof issues is Array
+})
+```
+
 ## TODOs: What's coming up next
 `good-first-issue` is still in an early state. I wanted to get `good-first-issue node` out the door, but have some other things I'm planning on implementing. Here's a list:
 
 - [x] `good-first-issue node` command
-- [ ] Interactive selector when `good-first-issue` is run without a sub command
+- [x] Interactive selector when `good-first-issue` is run without a sub command
 - [ ] Export good first issues for all repos that have a command
 - [ ] Add tests
 
 If you'd like to help with any of these, feel free to submit a PR or ask how you can help 🤗
 
 ## Projects
-The list of projects which are _currently_ supported.
-
-### Node.js
-The Node.js project spans over 100 repositories. Using `good-first-issue` to look for good first issues in Node.js will search through the entire [Node.js GitHub organization](https://github.com/nodejs/) including both technical and community repositories.
-
-#### Usage
-Via npx:
-```
-npx good-first-issue node
-```
-
-As a global module:
-```
-npm i -g good-first-issue
-good-first-issue node
-```
-
-### Electron
-Electron is one of the most vital pieces of desktop application infrastructure in the world, powering applications like Spotify, Slack,  Atom, VS Code, GitHub Desktop, WordPress.com, and hundreds of other applications. You can use `good-first-issue` to look for good first issues in the various Electron repositories in the [Electron GitHub organization](https://github.com/electron/).
-
-#### Usage
-Via npx:
-```
-npx good-first-issue electron
-```
-
-As a global module:
-```
-npm i -g good-first-issue
-good-first-issue electron
-```
-
-### VS Code
-VS Code is a text editor from Microsoft that has gained widespread adoption. It's built using JavaScript, Node.js, Electron, and a suite of other web technologies. You can use `good-first-issue` to look for good first issues in the [Microsoft/vscode repository](https://github.com/microsoft/vscode).
-
-#### Usage
-Via npx:
-```
-npx good-first-issue vscode
-```
+The table of projects which are _currently_ supported.
 
-As a global module:
-```
-npm i -g good-first-issue
-good-first-issue vscode
-```
-
-### Gutenberg
-Gutenberg is the new editor from the WordPress team, using a modern JavaScript front end stack. You can use `good-first-issue` to look for good first issues in the [WordPress/gutenberg repository](https://github.com/wordpress/gutenberg)
-
-#### Usage
-Via npx:
-```
-npx good-first-issue gutenberg
-```
-
-As a global module:
-```
-npm i -g good-first-issue
-good-first-issue gutenberg
-```
+| Name | Project `<project>` |
+| --- | --- |
+| Node.js | `node` |
+| Electron | `electron` |
+| VS Code | `vscode` |
+| Gutenberg | `gutenberg` |
 
```

Comment:
Since there was a common structure that we were previously including that we're now missing, it  would be good to include that in a more abstract way. Here's an example: https://github.com/bnb/good-first-issue#usage-1

Reply:
i dont get it as such

Reply:
Nevermind, it's already here: https://github.com/bnb/good-first-issue/pull/12/files#diff-04c6e90faac2675aa89e2176d2eec7d8R6